implants/imix/src/lib.rs

```diff
@@ -1,57 +1,27 @@
+#![deny(warnings)]
+#![allow(dead_code)]
```

Comment:
this should be on the code block with the fake dead code with a comment instead of for the whole file.

Reply:
I think it needs to be applied to almost every block in lib.rs because of how starlark bindings are a lil jank.

Reply:
Would you still want it if it's every single binding?

Reply:
yes

Reply:
Okey. Will do.

Reply:
<img width="1311" alt="image" src="https://github.com/spellshift/realm/assets/7121375/a4da8b98-9ab3-4c82-a266-8ac6b49c6497">
I think these will all need to be supressed.

Reply:
Resloved these issues by making agent a `pub mod`!
https://www.reddit.com/r/rust/comments/2dt2v6/why_does_rustc_complain_about_dead_code_when_im/